packages/api-server/src/handler.ts

```diff
@@ -110,33 +110,31 @@ export const webServerHandler = ({ port, socket, apiHost }: WebServerArgs) => {
   // Construct the graphql url from apiUrl by default
   // But if apiGraphQLUrl is specified, use that instead
   const graphqlEndpoint = coerceRootPath(
-    getConfig().web.apiGraphQLUrl ?? `${getConfig().web.apiUrl}graphql`
+    getConfig().web.apiGraphQLUrl ?? `${apiUrl}/graphql`
   )
 
-  let app = createApp()
+  const fastifyInstance = createApp()
 
-  // Attach middleware
-  // We need to proxy api requests to prevent CORS issues
   if (apiHost) {
-    app = withApiProxy(app, {
-      apiHost,
-      apiUrl,
-    })
+    // @TODO
+    // Attach middleware
```

Comment:
Generally fastify uses plugins, not middleware, and I don't think these can be attached after app has already started.